### PR TITLE
Updated o-personal-details to match latest design (#601)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
#601 

Closes #

### Short Description and Why It's Useful
Updated o-personal-details to match latest design

### Screenshots of Visual Changes before/after (If There Are Any)
o-personal-details-desktop-before
o-personal-details-desktop-after
o-personal-details-mobile-before
o-personal-details-mobile-after

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: components/organisms/o-personal-details.vue [hunk 1]
```diff
@@ -123,214 +121,201 @@
...
     </div>
   </div>
 </template>
 <script>
-import { required, minLength, email, sameAs } from 'vuelidate/lib/validators';
-import { PersonalDetails } from '@vue-storefront/core/modules/checkout/components/PersonalDetails';
-import { SfInput, SfButton, SfHeading, SfCheckbox, SfCharacteristic } from '@storefront-ui/vue';
-import { ModalList } from 'theme/store/ui/modals'
-import { mapActions } from 'vuex';
+import { required, minLength, email, sameAs } from "vuelidate/lib/validators";
```
Comment (dixitdeepak) [inline components/organisms/o-personal-details.vue:147]: These are the formatting changes, Please do not mix the formatting changes with functional change.s

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,13 +8,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.0.5] - ???
 
 ### Added
+
 - Added `VueObserveVisibility` and `VueLazyload` dependency
 
+### Changed / Improved
+
+- Updated o-personal-details to match latest designs(#601)
```
Comment (dixitdeepak) [inline CHANGELOG.md:16]: I think it'd be good if we mentioned what exactly updated

File: components/organisms/o-personal-details.vue (hunk 1 above)
Comment (azkyakhan) [reply components/organisms/o-personal-details.vue:147]: Ok will fix this

File: CHANGELOG.md (hunk 2 above)
Comment (azkyakhan) [reply CHANGELOG.md:16]: Ok

**Comment by dt2patel** [comment]
Looks good to me. A HTML and CSS refactor to help readability and customization. 

@Fifciu have a look, this also brings the design up to date with the latest Figma file (linked in issue)

**Review by dt2patel — APPROVED** [review]

**Pull request merged**